src/Machine.js

```diff
@@ -222,6 +222,9 @@ class Machine {
       this._includeMacro(macro, context, buffer);
     } else {
       // source inclusion
+
+      // replace for compatibility with Windows
+      instruction.value = instruction.value.replace(/\\/g, "/");
```

Comment:
We can't do it after evaluating cose evaluating breakes string. In evaluate function jsep parse backslashes into spec chars.